reals_check/test/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+#Copyright (c) Microsoft. All rights reserved.
+#Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+add_subdirectory(reals)
+add_subdirectory(both)
+
+set(reals_check_test_c_files
+    ./src/dummy.c
+)
+
+set(reals_check_test_h_files
+    ./inc/reals_check_test/dummy.h
+)
+
+add_library(reals_check_test ${reals_check_test_c_files} ${reals_check_test_h_files})
+target_include_directories(reals_check_test PUBLIC $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/reals_check/test/inc>)
+
+add_test(NAME RUN_REALS_CHECK_TEST COMMAND powershell ${CMAKE_CURRENT_SOURCE_DIR}/test_reals_check.ps1 ${CMAKE_CURRENT_SOURCE_DIR}/../reals_check.ps1 ${CMAKE_BINARY_DIR} $<CONFIG>)
```

Comment:
I understand this test is testing very targeted at the produced libs here (reals_check_test, reals_check_test_reals and reals_check_test_both).

However, there is no exercise of the code (note: line 33 of root CMakeLists.txt) that gathers all the lib with their paths and automatically runs the tool. 

I believe this can be done maybe in the yml of the project? I would generate it simply with -Drun_reals_check:BOOL=ON

Reply:
This is all looking promising, keep at it! I would like to see last iteration before I sign off.

Reply:
I've added a test to make sure that the correct libs are being selected for reals checking. 

Also adding the reals check target to run the check automatically.